packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -303,6 +399,23 @@ export class Table extends Construct {
             });
         }
     }
+
+    private registerNonKeyAttributes(nonKeyAttributes: string[]) {
```

Comment:
Rename to validateNonKeyAttribute and add docs

Reply:
Sure. I will.